### PR TITLE
man: further shorten print-unit-path example

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

**Comment by keszybz** [comment]
Hmm, but this example is in the man page for `sd_bus_message_new_method_call()`. `sd_bus_call_method()` is cool, but isn't it better to keep the example as it was?

Instead, maybe add the example using  `sd_bus_call_method()` to   `sd_bus_call_method(3)`?

**Comment by 1awesomeJ** [comment]
Good afternoon Mr Bluca.
My PR is failing lots of tests.
Coming here now, it looks like the failures may not be due to my code?

File: man/print-unit-path.c [hunk 1]
```diff
@@ -26,12 +26,16 @@ static int log_error(int error, const char *message) {
   return error;
 }
 
-static int print_unit_path(sd_bus *bus) {
-  _cleanup_(sd_bus_message_unrefp) sd_bus_message *m = NULL;
+int main(int argc, char **argv) {
   _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
-  _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+  _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL, *m = NULL;
+  _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
```
Comment (keszybz) [inline man/print-unit-path.c]: The cleanup order is confusing here. We allocate bus first, so we should free it last. Right now this happens in reverse order. Please move the definition of `bus` up.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: man/print-unit-path-call-method.c [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+  _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+  _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
+  int r;
+
+  r = sd_bus_open_system(&bus);
+  if (r < 0)
+    return log_error(r, "Failed to acquire bus");
+
+  r = sd_bus_call_method(bus, DESTINATION, PATH, INTERFACE,  MEMBER, &error, &reply, "u", (unsigned) getpid());
```
Comment (YHNdnzj) [inline man/print-unit-path-call-method.c]:
```suggestion
  r = sd_bus_call_method(bus, DESTINATION, PATH, INTERFACE, MEMBER, &error, &reply, "u", (unsigned) getpid());
```

**Pull request merged**